### PR TITLE
{lib}[intel/2015a] lxml 3.4.2 (OK)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/94234988301c33e2d99a for a full test report.

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/l/lxml/lxml-3.4.2-intel-2015a-Python-2.7.9.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+pylibdir = "lib/python%s/site-packages/%%(name)s" % pyshortver
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': [('%s-%%(version)s-py%s.egg' % (pylibdir, pyshortver),
+              '%s-%%(version)s-py%s-linux-x86_64.egg' % (pylibdir, pyshortver))],
+}
+
+moduleclass = 'lib'
+
```
Comment (wpoely86) [inline easybuild/easyconfigs/l/lxml/lxml-3.4.2-intel-2015a-Python-2.7.9.eb]: drop newline

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**